android/gradle-maven-push.gradle

```diff
@@ -0,0 +1,62 @@
+apply plugin: 'com.github.dcendents.android-maven'
+apply plugin: 'maven-publish'
+
+Properties localProps = new Properties()
+def localPropsFile = rootProject.file('local.properties')
+if (localPropsFile.exists()) {
+    localProps.load(localPropsFile.newDataInputStream())
+}
+
+task sourceJar(type: Jar) {
+    from android.sourceSets.main.java.srcDirs
+    classifier "source"
+}
+
+afterEvaluate { project ->
+
+    project.ext.getJFrogPassword = {
+        return System.getenv('JFROG_PASSWORD') ?: localProps.getProperty("jfrog_password")
+    }
+
+    project.ext.getJFrogUsername = {
+        return System.getenv('JFROG_USERNAME') ?: localProps.getProperty("jfrog_username")
+    }
+
+    publishing {
+        publications {
+            maven(MavenPublication) {
+                groupId group
+                artifactId project.name
+                version version
+                description description
+
+                artifact bundleReleaseAar
+                artifact sourceJar
+
+                pom.withXml {
+
+                    def dependenciesNode = asNode().appendNode('dependencies')
```

Comment:
this can be moved to a function.

Reply:
It can, but I don't see the benefit, I don't think we are going to use this anywhere else. Can you explain why you think we should do that?